wazuh/wazuh-dashboard-plugins
# Fix modules redirects with pre-defined filters

### Description
This PR adds the functionality to add filters via URL.

### Test

- Go to Server Management > Rules
- Click on 'View alerts of this Rule' 
- Check the filter is applied after redirects
- Navigate to another modules and check that the filters disappear (prevents the error when the index pattern changes and the field doesn't exists)
 

https://github.com/wazuh/wazuh-dashboard-plugins/assets/6089438/03b7438e-e2cb-47bd-b3d3-b654aab9ba5a



### Check List
